.github/workflows/deploy-daac.yml

```diff
@@ -44,7 +44,7 @@ jobs:
               job_spec/AUTORIFT.yml
               job_spec/INSAR_GAMMA.yml
               job_spec/RTC_GAMMA.yml
-              job_spec/INSAR_ISCE.yml
+              job_spec/INSAR_ISCE_TEST.yml
```

Comment:
It looks like this change is confined to the EDC-UAT deployment, but I just want to double-check with @jhkennedy that this change won't impact any of our other deployments